das_getter.py

```diff
@@ -0,0 +1,305 @@
+import httplib
+import os
+import json
+
+def cmsweb_get(url):
+    headers = {'Content-Type': 'application/json',
+               'Accept': 'application/json'}
+    cert = os.getenv('X509_USER_PROXY')
+    key = os.getenv('X509_USER_PROXY')
+    print('Using certificate and key from %s' % cert)
+    if cert is None:
+        print('No X509_USER_PROXY found! Run "voms-proxy-init -voms cms; export X509_USER_PROXY=$(voms-proxy-info --path)" and try again')
+        return {}
+
+    conn = httplib.HTTPSConnection('cmsweb.cern.ch', cert_file=cert, key_file=key)
+    conn.request("GET", url, headers=headers)
+    response = conn.getresponse()
+    status, data = response.status, response.read()
+    conn.close()
+    print('HTTP status: %s' % (status))
+    jsonfile = json.loads(data)
+    print json.loads(data)
+    filesizetotal=jsonfile[0]['file_size']
+    try:
+        #return json.loads(data)
+        return filesizetotal
+    except:
+        print('Error parsing JSON from:\n\n%s' % (data))
+
+
+Files = [
+'/ZeroBias/Run2017A-TkAlMinBias-PromptReco-v1/ALCARECO',
```

Comment:
Why there are so many files, I think few dataset names would be sufficient as an example. Also, it looks like they're repeating themselves